### PR TITLE
Follow user and create activities for Lesson and Relationship

### PR DESCRIPTION
**Pull request by QuangTranXuan**
Follow user and create activities for Lesson and Relationship

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -46,6 +46,20 @@ def destroy
     redirect_to users_url
   end
 
+  def following
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: RESTFUL

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -46,6 +46,20 @@ def destroy
...
+    @title = 'Following'
+    @user = User.find params[:id]
+    @users = @user.following.paginate page: params[:page], per_page: 10
+    render 'show_follow'
+  end
+
+  def followers
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: RESTFUL

File: app/controllers/followers_controller.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class FollowersController < ApplicationController
+  def index
+    @title = 'Followers'
+    @user = User.find params[:user_id]
```
Comment (VoTaiTri) [inline app/controllers/followers_controller.rb]: :id instead :user_id

File: app/controllers/followings_controller.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class FollowingsController < ApplicationController
+  def index
+    @title = 'Following'
+    @user = User.find params[:user_id]
```
Comment (VoTaiTri) [inline app/controllers/followings_controller.rb]: :id instead :user_id

File: app/controllers/followers_controller.rb (hunk 3 above)
Comment (QuangTranXuan) [inline app/controllers/followers_controller.rb]: No it's :user_id

File: app/controllers/followings_controller.rb (hunk 4 above)
Comment (QuangTranXuan) [inline app/controllers/followings_controller.rb]: No, it's :user_id

**Pull request closed without merging**